crates/agglayer-storage/src/stores/state/mod.rs

```diff
@@ -358,10 +369,7 @@ impl StateStore {
         Ok(())
     }
 
-    fn read_local_exit_tree(
-        &self,
-        network_id: NetworkId,
-    ) -> Result<Option<LocalExitTree>, Error> {
+    fn read_local_exit_tree(&self, network_id: NetworkId) -> Result<Option<LocalExitTree>, Error> {
```

Comment:
Nit: do you have some different formatting settings?

Reply:
I don't think so, this update respects the `rustfmt` config that we're using tho